src/libraries/Microsoft.VisualBasic.Core/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,28 +1,4 @@
-Compat issues with assembly Microsoft.VisualBasic:
-
-# Loaded via reflection so needs to be public in the implementation but we don't want to expose them publicly in the ref assembly:
```

Comment:
Are these changes intentional? What is the reasoning?

Reply:
Sort of, yeah. Not really related to this PR in particular but when adding the annotations on the src I started getting new APICompat warnings on since they didn't match the ref (as I hadn't updated that yet at that point) so I regenerated this file a few times until finally I regenerated it one last time after adding the changes back to the ref assembly again. Long story short, these baselines had gone way out of date since we don't regenerate them regularly, so this is just the result of regenerating them today. Of course I could remove this change from the PR and regenerate them outside of this effort if preferred.